MeetupPlatformApi/DataTransferObjects/UserCredentialsChangeDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.DataTransferObjects
+{
+    public class UserCredentialsChangeDto
```

Comment:
Please, add `[MaxLength]` validation for `Username` and `NewPassword` fields.
This data will go to the database, hence it should be validated.